lms/djangoapps/mobile_api/course_info/views.py

```diff
@@ -26,6 +28,8 @@
 User = get_user_model()
 log = logging.getLogger(__name__)
 
+UserType = Union[django.contrib.auth.models.User, django.contrib.auth.models.AnonymousUser]
```

Comment:
Thanks for the suggestion, at this line I define `UserType` by listing all classes for a `User`. This variable is used only for typing purposes and don't incorporated in any logic.
Also currently, it is not possible to use `get_user_model()` with MyPy due to it dynamic return value, it's necessary to import and specify class explicitly.
ref. https://www.github.com/typeddjango/django-stubs/issues/599

I would prefer leaving `django.contrib.auth.models.User` model, and in the future extend the `UserType` union with the new class model when it would be required for typing purposes.